### PR TITLE
LPS-136040

### PR DESCRIPTION
**Pull request by wanderlast**
https://issues.liferay.com/browse/LPS-136040

From @Odrakir1:

> This issue was caused by a missing group id, in this case, the Global group id. The asset was being fetched by the Control Panel's group id only, adding the Global group id in the search solved the problem.

### CODE REVIEW

**Comment by wanderlast** [comment]
ci:test:sf

**Review by drewbrokke — APPROVED** [review]

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/categorization.jsp [hunk 1]
```diff
@@ -46,7 +47,7 @@ AccountEntryDisplay accountEntryDisplay = (AccountEntryDisplay)request.getAttrib
 		<liferay-asset:asset-tags-selector
 			className="<%= AccountEntry.class.getName() %>"
 			classPK="<%= accountEntryDisplay.getAccountEntryId() %>"
-			groupIds="<%= new long[] {controlPanelGroup.getGroupId()} %>"
+			groupIds="<%= new long[] {controlPanelGroup.getGroupId(), siteGroup.getGroupId()} %>"
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/categorization.jsp:50]: @wanderlast you can just call `themeDisplay.getSiteGroupId()` here and remove the variable.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@wanderlast there are some CI failures, but after another look at this there is one small improvement we can make. See my inline comment.

Please update the code, then rebase on upstream and force-push. Thank you!

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/categorization.jsp (hunk 1 above)
Comment (wanderlast) [reply modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/categorization.jsp]: @drewbrokke done, made the change and rebased to upstream

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**